modules/serverless-workflow-basic.adoc

```diff
@@ -70,7 +94,7 @@ Address:    http://hello.default.svc.cluster.local
 
```

Comment:
URL: `http://hello.default.apps-crc.testing`

I don't see an Address field at all with the current "kn"